pkgs/development/libraries/mapnik/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl
+, boost, cairo, freetype, gdal, harfbuzz, icu, libjpeg, libpng, libtiff
+, libwebp, libxml2, proj, python, scons, sqlite, zlib
+}:
+
+stdenv.mkDerivation rec {
+  name = "mapnik-${version}";
+  version = "3.0.9";
+
+  src = fetchurl {
+    url = "https://mapnik.s3.amazonaws.com/dist/v${version}/mapnik-v${version}.tar.bz2";
+    sha256 = "1nnkamwq4vcg4q2lbqn7cn8sannxszzjxcxsllksby055d9nfgrs";
+  };
+
+  nativeBuildInputs = [ python scons ];
+
+  buildInputs =
+    [ boost cairo freetype gdal harfbuzz icu libjpeg libpng libtiff
+      libwebp libxml2 proj python sqlite zlib
```

Comment:
Does mapnik itself really need Python?  And does `python-mapnik` work with all interpreter versions?

If one would now create an environment with `python-mapnik`, then two different Python versions might be in the same environment.
cc @domenkozar